flake.d/README.md

```diff
@@ -0,0 +1,74 @@
+This directory contains support files for `flake.nix`, which is packaging to allow LMQL and its dependencies to be installed anywhere [Nix](https://nixos.com/) is available, even if Python or Node.js wasn't previously available at all!
```

Comment:
The link is very likely incorrect. Should probably be: https://nixos.org

Reply:
Oops; quite right, of course.

Reply:
Addressed.